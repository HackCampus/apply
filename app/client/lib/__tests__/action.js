const test = require('ava')

const action = require('../action')

test('action', t => {
  t.snapshot(action('foo'))
  t.snapshot(action('foo', {bar: 'baz'}))
})
