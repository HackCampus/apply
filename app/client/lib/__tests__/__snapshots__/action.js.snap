exports[`action 1`] = `"{\"type\":\"foo\"}"`;

exports[`action 2`] = `"{"type":"foo","payload":{"bar":"baz"}}"`;
